lib/rawHttpRequest/rawHttpRequestSchema.js

```diff
@@ -0,0 +1,201 @@
+module.exports = {
+
+	title: 'Raw HTTP Request (Advanced)',
+
+	globals: false,
+
+	input: {
+
+		method: {
+			type: 'string',
+			description: 'The HTTP verb to perform the request with',
+			enum: [
+				'GET',
+				'POST',
+				'PATCH',
+				'PUT',
+				'DELETE',
+				'HEAD',
+				'OPTIONS'
+			],
+		},
+
+		url: {
+			description: 'The URL to make the request with.',
+			oneOf: [
+
+				{
+					title: 'Endpoint',
+					type: 'object',
+					properties: {
+
+						endpoint: {
+							type: 'string',
+							description: 'The endpoint to call in relation to the base URL used by the connector.',
```

Comment:
maybe add an example of what the input looks like in this description?

Reply:
also if they pass a full url into this, what will happen? will it do the normal falafel thing and call that url? this seems like it would be unexpected behaviour

Reply:
I've added input validation for URL